### ISSUE TITLE
Soroban fee estimator 

### ISSUE THREAD

**Issue by kalepail** [stellar/laboratory#747]
As I've been spending a lot of time getting to understand fee calculations and am trying to understand our own explanation of the current state of fee affairs (https://github.com/stellar/soroban-docs/issues/760) I'm finding myself really wanting a fee calculator where you can arbitrarily enter different ledger values and then see what the correlated current fee for all three networks would be.

So current ledger size at {x}, I want to write {y} bytes of temporary data, it would cost {z} XLM. So more of a knowledge tool than transaction simulation. Really aiming to help folks visualize how fees are calculated in different scenarios. 

Basically just taking the limits table here https://soroban.stellar.org/docs/reference/resource-limits-fees#resource-limits and current fees table here https://soroban.stellar.org/docs/reference/resource-limits-fees#resource-fees and building a simple GUI out of it for fiddling with the what ifs of both.

**Comment by janewang** [comment]
This is a good idea. However, I think some parts of the fees like inclusion fee may need to exist on the RPC first. Also simulateTransaction may need to some additional design work. We can then expose the data on Lab. Adding this for v2 but need more work on the various dependencies, in particular the RPC
